experiment/experiment.go

```diff
@@ -127,10 +128,7 @@ func (e *Experiment) newMeasurement(input string) model.Measurement {
 	}
 }
 
-// Measure performs a measurement with the specified input. Note that the
-// returned measurement will have all its privacy sensitive values set
-// to their real values. It will be your responsibility to strip them and
-// use the default values if the user told us to do so.
+// Measure performs a measurement with the specified input.
```

Comment:
I feel better now that we're safe by default rather than depending on people reading the docs.